template/koa-rest-api/gantry.apply.yml

```diff
@@ -21,6 +22,13 @@ env:
 datadogSecretId: '{{values "datadogSecretId"}}'
 {{end}}
 
+openTelemetry:
+  enabled: {{.Values.openTelemetry.enabled | default true}}
+  {{if .Values.isProduction}}
```

Comment:
There's a convention currently with this naming. Assuming it'd be great to keep it that way.